siliconcompiler/report/dashboard/cli/board.py

```diff
@@ -343,8 +342,6 @@ def _init_singleton(self):
             auto_refresh=True
         )
 
-        atexit.register(self._stop_on_exit)
```

Comment:
without this, what ensures the live screen is safely terminated?

Reply:
That would be the atexit from the core.py